src/node/desktop/src/assets/locales/fr.json

```diff
@@ -0,0 +1,113 @@
+{
+  "common": {
+    "buttonOk": "OK",
+    "buttonCancel": "Cancel",
+    "buttonBrowse": "Browse...",
+    "unknownErrorOccurred": "An unknown error occurred."
+  },
+  "contextMenu": {
+    "saveImageAsDots": "Save image as...",
+    "saveImageAs": "Save image as",
+    "copyImage": "Copy Image",
+    "reload": "Reload",
+    "copyImageAddress": "Copy Image Address",
+    "cut": "Cut",
+    "copy": "Copy",
+    "paste": "Paste",
+    "selectAll": "Select All",
+    "inspectElement": "Inspect Element",
+    "errorDownloadingImage": "Error downloading image",
+    "downloadCancelledMessage": "The download was cancelled.",
+    "downloadInterruptedMessage": "The download was interrupted.",
+    "save": "Save"
+  },
+  "chooseRDialog": {
+    "documentTitle": "Choose R Installation",
```

Comment:
We will get this file localized before the release. In the meantime, let's change a few of these strings so we can tell if the "French" ones are being used (for testing of real builds). Especially for the "other" windows like "Choose R" and the Error page. Something like "FR Choose R Installation FR".

Reply:
Done on [b155366](https://github.com/rstudio/rstudio/pull/11062/commits/b155366f62316119db252025d40652f3c3db7e26)